Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.m

```diff
@@ -168,34 +167,16 @@ - (id)initWithPredicate:(NSPredicate *)predicate
 
 #pragma mark - Instance Methods
 
-- (void)reloadFetchedResults:(NSNotification *)note
-{
-    VOK_CDLog(@"NSNotification: Underlying data changed ... refreshing!");
-    NSError *error = nil;
-    if (![self.fetchedResultsController performFetch:&error]) {
-        VOK_CDLog(@"Unresolved error %@, %@", error, [error userInfo]);
-        abort();
-    }
-}
-
 - (void)reloadData
 {
     NSError *error = nil;
     if (![self.fetchedResultsController performFetch:&error]) {
-        VOK_CDLog(@"Unresolved error %@, %@", error, [error userInfo]);
-        abort();
+        NSAssert(NO, @"Unresolved error %@, %@", error, [error userInfo]);
     }
-    //FOR TESTING ONLY, NOT NECESSARY
-    [self.tableView reloadData];
 }
```

Comment:
This certainly predates your changes, but it feels to me like this method should be `- (BOOL)reloadDataError:(NSError **)error` and then maybe have `- (void)reloadData` as a convenience wrapper calling `reloadDataError:`.